### PR TITLE
Added get_bus_index to AudioServer

### PR DESCRIPTION
**Pull request by MadEqua**
Implements proposed enhancement #9171

### CODE REVIEW

File: servers/audio_server.cpp [hunk 1]
```diff
@@ -515,6 +515,15 @@ String AudioServer::get_bus_name(int p_bus) const {
 	return buses[p_bus]->name;
 }
 
+int AudioServer::get_bus_index(const StringName &p_bus_name) const {
+
+	if (bus_map.has(p_bus_name)) {
+		Bus *bus = bus_map[p_bus_name];
+		return buses.find(bus);
```
Comment (reduz) [inline servers/audio_server.cpp]: this is probably redundant, just do a for() int the buses to find the right one..

**Review by reduz — CHANGES_REQUESTED** [review]

**Comment by MadEqua** [comment]
You're right, was trying to avoid a loop of string comparisons, but just noticed StringName is pretty efficient at that.

File: servers/audio_server.cpp [hunk 2]
```diff
@@ -515,6 +515,16 @@ String AudioServer::get_bus_name(int p_bus) const {
 	return buses[p_bus]->name;
 }
 
+int AudioServer::get_bus_index(const StringName &p_bus_name) const {
+
+	for (int i = 0; i < buses.size(); ++i) {
+		if (buses[i]->name == p_bus_name) {
+			return i;
+		}
+	}
```
Comment (vnen) [inline servers/audio_server.cpp]: There's a trailing space here, clang-format isn't happy.

**Comment by vnen** [comment]
You have two unrelated commits in the PR, you should rebase your branch to remove them.

**Pull request closed without merging**